ckanext/canada/schemas/presets.yaml

```diff
@@ -210,10 +210,10 @@ presets:
     fluent_form_label:
       en:
         en: Contributor (English)
-        fr: Contributeur (anglais)
+        fr: Contributor (Anglais)
```

Comment:
these are all (anglais) and (français) in the other fields not (Anglais) and (Français)